api/docs/samples.dox

```diff
@@ -118,6 +118,9 @@ produces a binary-format trace file.
 The sample <a href="https://github.com/DynamoRIO/dynamorio/tree/master/api/samples/hot_bbcount.c">hot_bbcount.c</a>
 uses the drbbdup extension to count the execution of hot basic blocks which have exceeded a given hit threshold.
 
+The sample <a href="https://github.com/DynamoRIO/dynamorio/tree/master/api/samples/opcode_count.cpp">opcode_count.cpp</a>
+takes an opcode as input and uses drmgr's instrumentation events to count the number of times the instruction is executed.
```

Comment:
"the instruction" sounds like a single specific instruction: reword to sthg like "instructions with that opcde are"

Reply:
Done.